inbm/dispatcher-agent/tests/unit/sota/test_downloader.py

```diff
@@ -110,3 +116,98 @@ def _build_mock_repo(num_files=0):
             for i in range(0, num_files):
                 mem_repo.add("test" + str(i + 1) + ".rpm", b"0123456789")
         return mem_repo
+
+    @patch('dispatcher.sota.downloader.read_oras_token', return_value="mock_password")
+    @patch('dispatcher.sota.downloader.oras_download')
+    def test_tiberos_download_successful(self, mock_download, mock_read_token) -> None:
+        self.release_date = self.username = None
+        password = "mock_password"
+        mock_url = canonicalize_uri("https://registry-rs.internal.ledgepark.intel.com/one-intel-edge/tiberos:latest")
+
+        assert TestDownloader.sota_instance
```

Comment:
in tests, use functions like `self.assertEqual` or `self.assertTrue` rather than bare `assert` for clearer test results

Reply:
Updated the unit test by checking the type of the instance.